utils/test-stack-remover.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# The CI for this repo can sometimes leave some stacks lying around - every couple of months we need to clean them up.
+# This script will delete any stacks that are in DELETE_FAILED state, that contain the word _test_ in their name, and
+# that are not autoscaling stacks (these are child stacks that will be deleted when their parent is deleted)
+
+# no -u, as we rely on the $ROLE_ARN variable being passed in
+set -eo pipefail
+
+aws cloudformation list-stacks \
+| jq -r ".StackSummaries[] | {StackName, StackStatus} | select(.StackStatus == \"DELETE_FAILED\").StackName" \
+| rg test \
+| rg -vi autoscaling \
+| sort | uniq \
+| xargs -t -I {} aws cloudformation delete-stack \
+  --stack-name {} \
+  --role-arn "$ROLE_ARN"
```

Comment:
Unfortunately, shfmt will now ask that it look like:
```suggestion
  | jq -r ".StackSummaries[] | {StackName, StackStatus} | select(.StackStatus == \"DELETE_FAILED\").StackName" \
  | rg test \
  | rg -vi autoscaling \
  | sort | uniq \
  | xargs -t -I {} aws cloudformation delete-stack \
    --stack-name {} \
    --role-arn "$ROLE_ARN"
```

Reply:
But a rebase is a good idea too.

Reply:
fixed! i also changed the `rg` calls to `grep` cals - it's more likely to be installed